### ISSUE TITLE
Generating Entity Content missing bundle question

### ISSUE THREAD

**Issue by mgrimard** [hechoendrupal/DrupalConsole#1493]
When I generate entity content, the 5th question is missing the actual question. Instead, it just shows what should be the path to the question.

```
drupal generate:entity:content
```

Results of bundle question:
```
commands.generate.entity.questions.has-bundles (yes/no)
```

**Comment by jmolivas** [comment]
Which Drupal Console question are you using ?, this must be fixed on the latest release v0.9.10

**Comment by mgrimard** [comment]
Yeah, I just realized 0.9.10 was released an hour ago. This is resolved.
